vertical-pod-autoscaler/pkg/utils/status/status_updater.go

```diff
@@ -14,48 +14,46 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package logic
+package status
 
 import (
 	"time"
 
-	"k8s.io/autoscaler/vertical-pod-autoscaler/pkg/utils/status"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/klog"
 )
 
-const (
-	updateInterval = 10 * time.Second
-)
-
-// StatusUpdater periodically updates Admission Controller status.
-type StatusUpdater struct {
-	client *status.Client
+// Updater periodically updates Admission Controller status.
+type Updater struct {
```

Comment:
Soooo we already have an updater in VPA :)
Can you think of a similar name that will not cause clashes?

Reply:
In normal cases the Updater will be referenced by the package name. Creating it looks like this:
```
status.NewUpdater(...)
```
So there should be no confusion there.